one-line-benchmark/README.md

```diff
@@ -0,0 +1,17 @@
+### One line benchmark
+*Run a set of benchmarks with a single command*
+
+
+<!---
+TODO: Change branch here and in olb.sh 
+--->
+
+```bash
+eval "curl 'https://raw.githubusercontent.com/t-young31/excalibur-tests/olb/one-line-benchmark/olb.sh' > olb.sh && . olb.sh"
+```
+
+
+#### Requirements
+- curl
+- bash
+- gcc (any version, which can be located by `spack`)
```

Comment:
You're missing also spack (and hence python, too), right?

Reply:
Installed via git and conda respectively [here](https://github.com/t-young31/excalibur-tests/blob/3d4043010e94af731a78c23c4e6d471f9c3dbe02/one-line-benchmark/olb.sh#L82)